sources/world/world_countries_v2.hjson

```diff
@@ -39,6 +55,16 @@
       name: name
       desc: Country name
     }
+    {
+      type: property
+      name: population
+      desc: Population
+    }
+    {
+      type: property
+      name: area
+      desc: Area
+    }
```

Comment:
```suggestion

```
We'll never support metric fields in older versions of Kibana that use this version of World Countries. So I think we can remove these fields from the schema.

Reply:
I'll remove the fields for the `geo.json` output, thanks